src/components/view/ExternalAccountsView/ExternalAccountsView.tsx

```diff
@@ -56,9 +56,7 @@ export default function (props: ExternalAccountsViewProps) {
             {props.presentation &&
                 <NavigationBar title={title}
                     showBackButton={props.presentation == "Push"}
-                    showCloseButton={props.presentation == "Modal"}
-                    onBack={props.onBack}
-                    onClose={props.onClose} />
+                    showCloseButton={props.presentation == "Modal"} />
```

Comment:
@chrisnowak 👆 ?

Reply:
Looks like those functions (`onBack` and `onClose`) can probably be removed from the properties above too.

There is only one usage of `onBack`, and I don't think it is needed anymore.

Reply:
Looks like this is still being used in https://github.com/CareEvolution/MyDataHelpsUI/blob/main/src/components/view/StandaloneHealthAndWellnessView/StandaloneHealthAndWellnessView.tsx#L189.

Reply:
Yes, but the `onBack` handler on the `ExternalAccountsViewProps` that is being used there is never actually being called by the `ExternalAccountsView`, so I think the "back" functionality is working because of this:  https://github.com/CareEvolution/MyDataHelpsUI/blob/main/src/components/view/StandaloneHealthAndWellnessView/StandaloneHealthAndWellnessView.tsx#L75

```
 MyDataHelps.back = back;
```

So I think we can remove those handler functions from the props, since they aren't being called, and just clean up the code above that (the code you linked) so that it no longer passes a handler that gets ignored.